doc/mongoc_advanced_connections.page

```diff
@@ -95,6 +95,34 @@ main (int   argc,
       <p>Connecting directly to a UNIX domain socket is not a common practice.</p>
     </note>
   </section>
+   
+  <section id="SSL">
+    <info><link type="guide" xref="index#advanced-connections" /></info>
+    <title>Connecting to a server that requires SSL</title> 
+<p>These are instructions for configuring TLS/SSL connections.</p>
+   <listing> 
+   <title>Basic Configuration</title>
+    <p>To run a server locally (on port 27017, for example):</p>
+    <screen><code>$ mongod --port 27017 --sslMode requireSSL --sslPEMKeyFile server.pem --sslCAFile ca.pem </code></screen>
+    <p>If you are using a version of the C Driver earlier than 1.2.0, add <code>/?ssl=true</code> to the end of a client URI.</p>
+    <screen><code>mongoc_client_t *client = NULL;
+client = mongoc_client_new ("mongodb://localhost:27017/?ssl=true");</code></screen>
+<p>If using version 1.2.0 or later, clients connect to servers with SSL even if <code>/?ssl=true</code> is omitted.</p></listing>
+   <listing><title>Specifying Client Certificates and CA Files</title>
+   <p>The C Driver can be configured to present a client certificate using a <code>mongoc_ssl_opt_t</code>:</p>
+<screen><code>const mongoc_ssl_opt_t *ssl_default = mongoc_ssl_opt_get_default ();
+mongoc_ssl_opts_t ssl_opts = { 0 };
+mongoc_client_t client = mongoc_client_new ("mongodb://localhost:27017");
+
+/* optional: copy in default trust directory and file, not implemented as of version 1.4.0 */
```

Comment:
This isn't correct; say something like "Optionally copy in a custom trust directory or file; otherwise the default is used."

These instructions will be released with version 1.4.0, which includes this fix:

https://jira.mongodb.org/browse/CDRIVER-1142